ynr/apps/elections/templates/elections/ballot_view.html

```diff
@@ -31,92 +31,323 @@
+          {% include 'candidates/_person_form.html' %}
+        {% endwith %}
+        </form>
+      </div>
+    {% endif %}
+
+
+    {% if membership_edits_allowed %}
+
+
+    {% comment %}<h2>People not standing again</h2>
+
+    {{ people_not_standing }}
+{% endcomment %}
+    {% if candidates_might_stand_again %}
+
+
+    <div class="candidates__previous">
+      <h3>Is a candidate from an earlier election standing again?</h3>
+
+
+      <table class="table candidates-list">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Party</th>
+            {% if user.is_authenticated %}
+            <th>Actions</th>
+            {% endif %}
+          </tr>
+        </thead>
+        <tbody>
+          {% for candidate in candidates_might_stand_again %}
+          <tr>
+            <td>
+              <a href="{{ candidate.person.get_absolute_url }}">
+                <img class="person-avatar" src="{{ candidate.person.get_display_image_url }}"/>
+                {% if candidate.result.is_winner %}<strong>{% endif %}
+                  {{ candidate.person.name }}
+                {% if candidate.result.is_winner %}</strong>{% endif %}
+              </a>
+            </td>
+            <td>{{ candidate.party.name }}</td>
+            <td>
+              <button class="button tiny js-toggle-source-confirmation standing">Standing again</button>
+              <button class="button tiny js-toggle-source-confirmation not-standing">Not standing again</button>
+              {% include 'candidates/_source_confirmation.html' with standing='standing' action='candidacy-create' person_id=candidate.person.pk %}
+              {% include 'candidates/_source_confirmation.html' with standing='not-standing' action='candidacy-delete' person_id=candidate.person.pk %}
+
+              <a href="{% url 'person-update' person_id=candidate.person.pk %}" class="button tiny secondary">
+                Edit
+              </a>
+            </td>
+          </tr>
+          {% endfor %}
+
+        </tbody>
+      </table>
     </div>
+    {% endif %}
+
+
+    {% if people_not_standing %}
+    <div class="candidates__not-standing">
+      <h3>These people from earlier elections are known not to be standing again</h3>
+      <table class="table candidates-list">
+        <thead>
+          <tr>
+            <th>Name</th>
+            {% if user.is_authenticated %}
+            <th>Actions</th>
+            {% endif %}
+          </tr>
+        </thead>
+        <tbody>
+          {% for person in people_not_standing %}
+          <tr>
+            <td>
+              <a href="{{ person.get_absolute_url }}">
+                <img class="person-avatar" src="{{ person.get_display_image_url }}"/>
+                {{ person.name }}
+              </a>
+            </td>
+            <td>
+              <a class="button tiny js-toggle-source-confirmation standing">Actually, they are standing!</a>
```

Comment:
Same issue here: A few lines up, we're only showing the "Actions" header if the user is authenticated, but then we're rendering the column/form content anyway, even if they're not.